distribution/docs/src/main/resources/content/_installing/solr-service.adoc

```diff
@@ -0,0 +1,75 @@
+:title: Run Solr as Managed Service
+:type: startingIntro
+:status: published
+:summary: Install Solr as a Service.
+:project: ${branding}
+:order: 05
+
+==== {title}
+
+These instructions are for configuring Solr as a service managed by the operating system.
+
+===== Configure Solr as a Windows Service
+
+Windows users can use the *Task Scheduler* to start Solr as a background process.
+
+. If ${branding} is running, stop it.
+. Edit `<${branding}_HOME>/etc/system.properties` and set `start.solr=false`.
+  This prevents the ${branding} scripts from attempting to manage Solr's lifecycle.
+. Start the *Windows Task Scheduler* and open the *Task Scheduler Library*.
+. Under the Actions pane, select *Create Basic Task...*.
+. Provide a useful name and description, then select *Next*.
+. Select *When the computer starts* as the Trigger and select *Next*.
+. Select *Start a program* as the Action and select *Next*.
+. Select the script to start Solr:
++
+-----
+${home_directory}\bin\ddfsolr.bat
+-----
++
+. Add the argument `start` in the window pane and select *Next*.
+. Review the settings and select *Finish*.
+
+It may be necessary to update the *Security Options* under the task *Properties* to
+*Run with highest privileges* or setting user to "SYSTEM".
+
+Additionally, the process can be set to restart if it fails. The option can be found in the
+ the *Properties* > *Settings* tab.
+
+Depending on the system it may also make sense to delay the process from starting for a
```

Comment:
❓Should we recommend that the delay by shorter than the one used for DDF?

Reply:
If there was a history of problems caused by users setting Solr timeout >= DDF timeout, I would definitely add it. Without evidence, I feel adding that to our documentation is not appropriate.